src/main/java/teammates/ui/controller/AdminSearchPageAction.java

```diff
@@ -10,6 +10,7 @@
 import teammates.common.datatransfer.InstructorAttributes;
```

Comment:
yes it should be done in an new action. Moreover, should we delete the original account entity if the account has no related student entity anymore after the reset (which should not be deleted if the account has already been registered with other "student"s )?